app/components/directives/settings/networkcreateform.ts

```diff
@@ -5,6 +5,7 @@
 import {Component, Input, Output, EventEmitter, OnInit, NgZone} from "@angular/core";
```

Comment:
If you run first run wizard once and setup network, logout, then remove the first run key from local storage and log back in, and run the wizard again, the data in the network form is pre-populated with the values from the last time. Why is that happening?